editor/src/core/model/conditionals.ts

```diff
@@ -168,6 +168,18 @@ export function maybeBranchConditionalCase(
   }
 }
 
+export function getConditionalCaseFromMetadata(
```

Comment:
This looks very similar to `getConditionalCase`, maybe the two can be unified?

Reply:
`getConditionalCase` seems to be unused, so I'm rolling it into `getConditionalCaseFromMetadata`

Reply:
I'd rather not touch the deprecated function (since that'll be refactored out anyway with time and I don't want to risk breaking unrelated functionality).

`getConditionalCase` is not used anywhere, and the problem it solves isn't relevent to `getConditionalCaseFromMetadata`: `getConditionalCase` was originally written to determine whether a given branch is the currently active branch of a conditional, and `getConditionalCaseFromMetadata` is to determine if a branch is in the true or the false branch. I deleted `getConditionalCase` in https://github.com/concrete-utopia/utopia/pull/3586/commits/8ef4a20560099baed38c631a1667698f286da3f7 and renamed `getConditionalCaseFromMetadata` to `getConditionalCaseCorrespondingToBranchPath`